views/repo_list.pug

```diff
@@ -1,9 +1,25 @@
 h1
   Github Repositories
 
-.git-repo-areas
+.git-repo-areas(ng-controller="ListItemCtrl")
```

Comment:
Becuase in the first time run ListItemCtrl I have executed the function $scope.getColors()
So if I put this ng-controller in ng-repeat it will executes the function $scope.getColors() multi-times.